payments-ui-core/res/values/strings.xml

```diff
@@ -93,4 +93,5 @@
   <string name="stripe_setup_button_label">Set up</string>
   <!-- Label for UPI ID number field on form -->
   <string name="stripe_upi_id_label">UPI ID</string>
+    <string name="stripe_paymentsheet_payment_method_multibanco">Multibanco</string>
```

Comment:
I think we normally put these in donottranslate, as most of the time these are names, and don't get translated.

Reply:
The lokalise build won't fail if you do that too. See others here: https://github.com/stripe/stripe-android/blob/c766796ffb43e8aee4f75330934b2940e2bbc4d0/payments-ui-core/res/values/donottranslate.xml#L31

Reply:
Oh nice! Android Studio warns against adding anything to anything but the translations editor, so I didn't realize some were in different files